plugins/node/opentelemetry-instrumentation-aws-sdk/package.json

```diff
@@ -75,7 +75,7 @@
     "sinon": "15.0.1",
     "test-all-versions": "5.0.1",
     "ts-mocha": "10.0.0",
-    "typescript": "4.3.4"
+    "typescript": "4.4.4"
```

Comment:
Hi. Why did you change the typescript version here if you split it to another PR #1616?

Reply:
I can revert this, but it will cause the current PR fail and unable to run unit test :(

Reply:
I can revert this, but it will cause the current PR fail and unable to run unit test :(

Reply:
understood. so maybe leave it as is.